.remarkrc.js

```diff
@@ -25,13 +25,24 @@ exports.plugins = [
   [require('remark-lint-prohibited-strings'), [2, [
     { no: 'master' },
     { no: 'slave' },
+    { no: 'blacklist' },
+    { no: 'whitelist' },
+    { no: 'graylist' },
+    { no: 'dark(-|\s)?greylist' },
+    { no: 'light(-|\s)?greylist' },
+    { no: 'tribal(-|\s)?knowledge' },
+    { no: 'tribal(-|\s)?wisdom' },
     { no: 'tribal' },
     { no: 'tribalism' },
+    { no: 'tribal(-|\s)?division'},
```

Comment:
```suggestion
    { no: 'tribal(-|\s|ism)?(knowledge|wisdom|division)?'},
```

Can this and line 33 and 34 be combined, like we do on line 68?